spec/collator.html

```diff
@@ -48,7 +46,7 @@ <h1>InitializeCollator ( _collator_, _locales_, _options_ )</h1>
         1. Set _opt_.[[localeMatcher]] to _matcher_.
         1. Let _collation_ be ? GetOption(_options_, *"collation"*, ~string~, ~empty~, *undefined*).
         1. If _collation_ is not *undefined*, then
-          1. If _collation_ does not match the Unicode Locale Identifier `type` nonterminal, throw a *RangeError* exception.
```

Comment:
In https://unicode-org.github.io/cldr-staging/ldml/v36/tr35.html#Unicode_language_identifier where the grammar of 'type' defined, it is part of the 'Unicode Locale Identifier' but it mention no 'language tag' in the grammar, so I think your change make it more confusing and prompt the users to ask, does it mean 
unicode_language_id OR unicode_language_subtag	 OR tlang? 

Notice "A Unicode locale identifier is composed of a Unicode language identifier" therefore, the type is actually not part of unicode_language_id but only part of keyword 
and therefore part of unicode_locale_extensions
and therefore part of extensions
and therefore part of unicode_locale_id.
but type is not part of unicode_language_id nor part of unicode_language_subtag	 nor part of tlang

therefore, your change make thing very confusing.

Reply:
As noted above, "language tag nonterminal" links to the definition of that term as "_defined in the grammars of [Section 3 Unicode Language and Locale Identifiers](https://unicode.org/reports/tr35/#Unicode_Language_and_Locale_Identifiers)_", so "_the `type` language tag nonterminal_" specifically refers to the nonterminal named `type` at that page, which expands to `alphanum{3,8} (sep alphanum{3,8})*`.